src/main/scala/nl.knaw.dans.easy.deposit/Errors.scala

```diff
@@ -109,7 +116,15 @@ object Errors extends DebugEnhancedLogging {
     extends ServletResponseException(BAD_REQUEST_400, s"ZIP file is malformed. $msgAboutEntry")
 
   case class PendingUploadException()
-    extends ServletResponseException(CONFLICT_409, "Another upload is pending. Please try again later.")
+    extends ServletResponseException(CONFLICT_409, "Another upload or submit is pending.")
+
+  case class NoStagingDirException(file: File)
+    extends ServletResponseException(INTERNAL_SERVER_ERROR_500, s"staging directory was not created: $file")
+
+  case class ClientAbortedUploadException(path: String)
+    extends ServletResponseException(OK_200, s"Client aborted upload of path $path") {
+    logger.info(getMessage)
```

Comment:
Probably this is already logged with the response (only not the body, because it's a 200).